app/main.py

```diff
@@ -16,4 +16,22 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"immutable": [],
+                    "mutable": []}
+
+
+def separate_mutable_immutable(*args) -> None:
+    for item in args:
+        if not isinstance(item, (list, set, dict)):
+            sorted_variables["immutable"].append(item)
+        else:
+            sorted_variables["mutable"].append(item)
+
+
+separate_mutable_immutable(lucky_number,
+                           pi,
+                           one_is_a_prime_number,
+                           name,
+                           my_favourite_films,
+                           profile_info,
+                           marks, collection_of_coins)
```

Comment:
In this task, you just need to fill the sorted_variables dictionary with the required values, no additional logic is required.